pkg/actors/actors.go

```diff
@@ -27,6 +27,8 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+const actorTypeIDSeparator = "$"
```

Comment:
an actor id can contain $ as well. Use some more complex actor separator something like __delimit__

Reply:
Let's discuss this since we might need to use some sort of encoding before concatenation.

Reply:
Do we document what characters can be in an actor id since it's simply a string?

Reply:
The question is whether to limit to to support extra character via encoding.

Reply:
Done. Please, check the latest version. I would like to wait for the new E2E test for actors to me merged first: #902